<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesDialog</class>
 <widget class="QDialog" name="PropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Properties</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="general_tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QFrame" name="line">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>41</y>
         <width>423</width>
         <height>16</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::HLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
      <widget class="QGroupBox" name="attributes_group">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>97</width>
         <height>82</height>
        </rect>
       </property>
       <property name="title">
        <string>Attributes</string>
       </property>
       <layout class="QVBoxLayout" name="attributes_layout">
        <item>
         <widget class="QCheckBox" name="readonly_cb">
          <property name="text">
           <string>Read-only</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="hidden_cb">
          <property name="text">
           <string>Hidden</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="accessed_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>161</y>
         <width>52</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Accessed:</string>
       </property>
      </widget>
      <widget class="QLabel" name="created_value">
       <property name="geometry">
        <rect>
         <x>68</x>
         <y>139</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="accessed_value">
       <property name="geometry">
        <rect>
         <x>68</x>
         <y>161</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="modified_value">
       <property name="geometry">
        <rect>
         <x>68</x>
         <y>117</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="created_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>139</y>
         <width>44</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Created:</string>
       </property>
      </widget>
      <widget class="QLabel" name="type_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>51</y>
         <width>28</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
      <widget class="QLabel" name="type_value">
       <property name="geometry">
        <rect>
         <x>68</x>
         <y>51</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="location_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>73</y>
         <width>48</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Location:</string>
       </property>
      </widget>
      <widget class="QLabel" name="location_value">
       <property name="geometry">
        <rect>
         <x>68</x>
         <y>73</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="size_value">
       <property name="geometry">
        <rect>
         <x>68</x>
         <y>95</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="size_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>95</y>
         <width>24</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Size:</string>
       </property>
      </widget>
      <widget class="QLabel" name="modified_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>117</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Modified:</string>
       </property>
      </widget>
      <widget class="QLabel" name="icon_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="name_label">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>10</y>
         <width>391</width>
         <height>31</height>
        </rect>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
